eth/client_ticketbroker.go

```diff
@@ -62,17 +62,18 @@ func (c *client) RedeemWinningTicket(ticket *pm.Ticket, sig []byte, recipientRan
 
 // GetSenderInfo returns the info for a sender
 func (c *client) GetSenderInfo(addr ethcommon.Address) (*pm.SenderInfo, error) {
-	info, err := c.TicketBrokerSession.GetSenderInfo(addr)
+	info, err := c.WrappedTicketBrokerSession.GetSenderInfo(addr)
```

Comment:
Not exactly possible because the higher level `sess.Contract.TicketBrokerCaller.contract.Call` method lives on an unexposed object. 

But I can do a low level call instead, just a bit longer.